tests/test_payloads.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import csv
+
+import pytest
+
+import garak._config
+import garak.payloads
+
+
+def test_payload_Manager():
+    f = garak.payloads.Manager()
+    assert isinstance(f, garak.payloads.Manager)
+
+
+def test_payload_list():
+    f = garak.payloads.Manager()
+    for payload_name in f.search():
+        assert isinstance(
+            payload_name, str
+        ), "Payload names from Manager().search() must be strings"
+
+
+PAYLOAD_NAMES = list(
+    garak.payloads.Manager().search()
+)  # default includes local custom payloads to help test them
```

Comment:
This code will be executed on import, this may result in brittle or missed results.

Reply:
Fair point. I haven't found a fixture-based solution to this yet. There's also something a bit circular about using the Manager class to get valid inputs for the Manager class, but that's a secondary point to your comment.

Reply:
sunk enough time into this, didn't find a resolution using a fixture - would like to have a fixture detemine the value at `session` scope, and then use this fixture to iterate, but did not succeed.